handsontable/.config/loader/sass-rtl-loader.js

```diff
@@ -0,0 +1,33 @@
+function scssRtl(source) {
+  return `
+@mixin load($direction: ltr) {
+  $inline-start: left;
+  $inline-end: right;
+  
+  @if $direction == rtl {
+    $inline-start: right;
+    $inline-end: left;
+  }
+
+${source}
+
+}
+
+@include load();
+
+[dir=rtl] {
+  @include load(rtl);
+}
+`;
+}
+
+module.exports =  function(source) {
+  
+  if(this.resource.endsWith('.scss')){
+    return scssRtl(source);
+  }
+  if(this.resource.endsWith('.sass')){
+    this.getLogger('sass-rtl-loader').warn('RTL features for SASS are not supported. Please convert to SCSS.');
+  }
```

Comment:
I like this check because the existence of `sass-loader` suggests that we handle `.scss` and `.sass` with the same level. It isn't true. The direction features are enabled only for `.scss` files.

It's also no cost to keep this if in a code, and it provides useful information.